tidy3d/components/data/sim_data.py

```diff
@@ -213,13 +213,103 @@ def at_centers(self, field_monitor_name: str) -> xr.Dataset:
 
         return monitor_data.colocate(**xyz_kwargs)
 
-    def get_intensity(self, field_monitor_name: str) -> xr.DataArray:
+    def get_poynting_vector(self, field_monitor_name: str) -> xr.Dataset:
+        """return `xarray.Dataset` of the Poynting vector at Yee cell centers.
+
+        Calculated values represent the instantaneous Poynting vector for time-domain fields and the
+        complex vector for frequency-domain: S = 1/2 E × conj(H).
+
+        Only the available components are returned, e.g., if the indicated monitor doesn't include
+        field component `"Ex"`, then `"Sy"` and `"Sz"` will not be calculated.
+
+        Parameters
+        ----------
+        field_monitor_name : str
+            Name of field monitor used in the original :class:`Simulation`.
+
+        Returns
+        -------
+        xarray.DataArray
+            DataArray containing the Poynting vector calculated at the center locations of the Yee
+            grid.
+        """
+
+        field_dataset = self.at_centers(field_monitor_name)
+
+        time_domain = isinstance(self.monitor_data[field_monitor_name], FieldTimeData)
+
+        poynting_components = {}
+
+        dims = "xyz"
+        for axis, dim in enumerate(dims):
+            dim_1 = dims[axis - 2]
```

Comment:
our convention to get the tangential field components is to "pop" the normal dimension (eg. for normal (x, y, z), the tangential components are usually (yz), (xz), (xy) in our code.

It seems here that if, eg `dim` is `y`, then the dim1 and dim2 are `z, x`, respectively. Just want to make sure / double check that this doesn't introduce an unexpected minus sign.

Reply:
This is on purpose, so that we get the correct sign on the Poynting vector: `Sy = Ez Hx - Ex Hz`